Cargo.toml

```diff
@@ -28,12 +28,12 @@ serde = { version = "1", features = ["derive"] }
 thiserror = "1"
 toml = "0.5"
 
-[dependencies.memmap]
-version = "0.7"
+[dependencies.memmap2]
+version = "0.3"
 optional = true
 
 [features]
-default = ["memmap"]
+default = ["memmap2"]
```

Comment:
This would break the 'memmap' feature gating that is used in some module. I propose that we add an indirection along the lines of:

```
[features]
default = ["memmap"]
memmap = ["memmap2"]
```

Reply:
Done.